src/api/java/jni/cvcJavaApi.h

```diff
@@ -0,0 +1,24 @@
+#ifndef CVC__JAVA_API_H
+#define CVC__JAVA_API_H
```

Comment:
Are we going to have more code in this file? We should use the usual namespaces and include guards here.

Reply:
Perhaps more cases need to be added to this file. What would be a nice namespace for the java api
Something like `cvc5::api::java` ? 
What do you mean by guards?